xclim/sdba/properties.py

```diff
@@ -0,0 +1,735 @@
+"""
+Properties submodule
+=================
+ Statistical Properties is the xclim term for 'indices' in the VALUE project.
+ SDBA diagnostic tests are made up of properties and measures.
+
+ This framework for the diagnostic tests was inspired by the VALUE project (www.value-cost.eu/).
```

Comment:
```suggestion
 This framework for the diagnostic tests was inspired by the [VALUE]_ project.
 
 .. [VALUE] www.value-cost.eu/.
```
So it shows as a reference+hyperlink in the docs.